### ISSUE TITLE
[Bug]:  change all the uses of channel_name now that  spikeinterface > 0.101.0 is released

### ISSUE THREAD

**Issue by tabedzki** [catalystneuro/neuroconv#1004]
### What happened?

[baserecordingextractorinterface.py](https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py#L38) has the following
` TODO remove this and go and change all the uses of channel_name once spikeinterface > 0.101.0 is released`

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by h-mayorquin** [comment]
Yeah, that if becomes unecessary after the latest release. Do you have any downstream problem that derives for this or just contributing to our code cleanliness : )? (thanks btw! just asking in case there is something urgen to fix here)

**Comment by tabedzki** [comment]
Nothing urgent on my end regarding this. I was exploring the code base to figure out how to implement the other issue about specific SpikeInterface extractor for NIDQ and came across this.

**Comment by h-mayorquin** [comment]
Thanks for opening an issue.
